id/mod.go

```diff
@@ -1,7 +1,60 @@
 package id
 
+import (
+	"crypto/sha256"
+	"go.dedis.ch/dela/mino"
+	"math/big"
+)
+
 type Id interface {
-	GetLength() byte
+	GetLength() int
```

Comment:
Consider removing the `Get` prefix here. It's idiomatic in Go to have a getter without the `Get` prefix in method names.

Reply:
Done, thank you! Does this also apply to `GetDigit`, which is not a simple getter?